main/lxc/lxc.initd

```diff
@@ -60,6 +69,9 @@ _autostart() {
 
 start() {
 	checkconfig || return 1
+	if [ -n "${systemd_container}" ]; then
```

Comment:
if you set `systemd_container=no` then will it still run systemd_ctr. Maybe do:

````sh
        if yesno "$systemd_container"; then
                systemd_ctr
        fi
````

And we should probably umount it on lxc stop?

Reply:
`yesno` is not available so I added a `case` statement instead

added an unmount to `stop()`